src/encoded/search.py

```diff
@@ -494,9 +495,9 @@ def prepare_search_term(request):
     Ignore certain keywords, such as type, format, and field
     """
     prepared_terms = {}
-    prepared_vals = []
-    for field, val in request.normalized_params.iteritems():
-        if field.startswith('validation_errors') or field.startswith('aggregated_items'):
+    for field, val in request.normalized_params.items():
+        if field.startswith('validation_errors') or field.startswith('aggregated_items') or \
```

Comment:
maybe factor this test out as a predicate (`is_additional_facet_name` or some such?)